rules/type-declaration/tests/Rector/ClassMethod/AddArrayReturnDocTypeRector/Fixture/a.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+namespace Rector\TypeDeclaration\Tests\Rector\ClassMethod\AddArrayReturnDocTypeRector\Fixture;
+
+abstract class A {
```

Comment:
Could you use more descriptive name, like `SkipShapeTypeWithParentDeclaratoin`?

The fixture is named afte this and it makes debugging months later much easier.